b2share/modules/communities/cli.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 University of Tuebingen, CERN.
+# Copyright (C) 2015 University of Tuebingen.
```

Comment:
This is a new file. There was no copyright in 2015. You can also add SURFsara in the list of organizations.